Sources/Controllers/2.0/Settings/NavigationSettings/OADefaultSpeedViewController.mm

```diff
@@ -88,20 +88,15 @@ - (void) generateData
             _ratio = 1;
             break;
     }
-    CGFloat settingsMinSpeed = self.appMode.getMinSpeed;
-    CGFloat settingsMaxSpeed = self.appMode.getMaxSpeed;
     if ([_speedParameters[@"defaultSpeedOnly"] boolValue])
     {
-        _minValue = round(1 * _ratio);
-        _maxValue = round(300 * _ratio);
+        _minValue = round(self.appMode.baseMinSpeed * _ratio);
```

Comment:
Min better to set to 1 as minumum
minValue = max(1, ceil(self.appMode.baseMinSpeed * _ratio));